app/mailers/asynchronous_notify.rb

```diff
@@ -0,0 +1,36 @@
+require 'celluloid'
+
+class AsynchronousNotify < ActionMailer::Base
```

Comment:
I really want to have this feature use Sidekiq. You can even just create a mailer, and then call it like this: `BroadcastMailer.delay.send_email` this will make use of Sidekiq's email delay system: https://github.com/gitlabhq/gitlabhq/pull/7918